pyquil/api/_compiler.py

```diff
@@ -122,6 +122,7 @@ def native_quil_to_executable(
             num_shots=nq_program.num_shots,
             quantum_processor_id=self.quantum_processor_id,
             translation_options=api_options or self.api_options,
+	    client=self._client_configuration,
```

Comment:
This needs to match the indentation of the other arguments, other than that, LGTM!